app/models/new_york_time.rb

```diff
@@ -1,7 +1,8 @@
 class NewYorkTime
+  BUSINESS_NEWS_ENDPOINT='https://api.nytimes.com/svc/topstories/v2/business.json?api-key='
 
   def self.get_business_news
-    response = HTTParty.get("https://api.nytimes.com/svc/topstories/v2/business.json?api-key=#{ENV['NY_KEY']}")
+    response = HTTParty.get("#{BUSINESS_NEWS_ENDPOINT}#{ENV['NY_KEY']}")
     data_to_json = JSON.parse(response.body, symbolize_names: true)[:results]
     # sort news array [latest ... oldest]
     data_to_json.sort { |data1, data2| Time.parse(data2[:created_date]).to_i <=> Time.parse(data1[:created_date]).to_i }
```

Comment:
I also noticed the test is only testing if news is `nil`, perhaps we should be testing if it's parsing and sorting the news correctly.